themes/root/templates/RecordDriver/AbstractBase/export-ris.phtml

```diff
@@ -149,12 +164,42 @@ if (is_array($notes)) {
     }
 }
 
+echo 'L4  - ' . $this->serverUrl($this->recordLinker()->getUrl($this->driver)) . "\r\n";
+
 foreach ($this->record($this->driver)->getUrlList() as $url) {
     echo 'UR  - ' . "$url\r\n";
 }
 
 if ($doi = $this->driver->tryMethod('getCleanDOI')) {
-    echo "DO  - $doi\r\n";
+    echo 'DO  - ' . strip_tags($doi) . "\r\n";
```

Comment:
I was not sure if that method was supposed to return HTML, although I did think it was not particularly "clean". In any case, it does return HTML (I forgot if it was with EDS or FOLIO), but I am also not sure if some calling code relies on it. It is used in HTML templates...

Reply:
The method is called `getCleanDOI` because it's supposed to return a raw DOI -- if it's returning something else, that's a bug. I would imagine this is probably related to the EDS code.

Reply:
OK. It is EDS. [Here](https://catalog.lib.msu.edu/EdsRecord/rzh,178847317#details) is an example ("DigiSpan: Development and Evaluation of a Computer-Based, Adaptive Test of Short-Term Memory and Working Memory."). The HTML is in Items - Data with Name: "DOI":
`&lt;link linkTarget=&quot;URL&quot; linkWindow=&quot;10.1044/2024_JSLHR-23-00466&quot; linkTerm=&quot;https://www.doi.org/10.1044/2024_JSLHR-23-00466&quot;&gt;10.1044/2024_JSLHR-23-00466&lt;/link&gt;`